Content.Shared/GameObjects/Components/Doors/SharedDoorComponent.cs

```diff
@@ -20,5 +20,6 @@ public enum DoorVisualState
         Open,
         Closing,
         Deny,
+        Light,
```

Comment:
Maybe call this something more descriptive? I think `Light` could mean a lot of things.